rust/docker-init.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+if [ $PACKAGE_NAME = "gateway" ]; then
+  IFACE="utun"
+  iptables -A FORWARD -i $IFACE -j ACCEPT; iptables -A FORWARD -o $IFACE -j ACCEPT; iptables -t nat -A POSTROUTING -o eth+ -j MASQUERADE
+fi
```

Comment:
Suggest keeping the conditional configurable at the service level:
```suggestion
if [ "$ENABLE_MASQUERADE" = "1" ]; then
  IFACE="utun"
  iptables -A FORWARD -i $IFACE -j ACCEPT; iptables -A FORWARD -o $IFACE -j ACCEPT; iptables -t nat -A POSTROUTING -o eth+ -j MASQUERADE
fi
```

Reply:
Ah, also don't forget ipv6 ;-)

```
ip6tables -A FORWARD -i $IFACE -j ACCEPT; ip6tables -A FORWARD -o $IFACE -j ACCEPT; ip6tables -t nat -A POSTROUTING -o eth+ -j MASQUERADE
```

Reply:
I will leave ipv6 for later, it still needs some tuning.